### PR TITLE
+1 icon

### PR DESCRIPTION
**Pull request by Ahmed-Khalifa**
### Added
Team Pixel Wallpapers (`com.lagguy.teampixelwallpapers/com.amaan.app.features.MainActivity`)

## Contributor's checklist
- [x] I followed [the Lawnicons guidelines](https://github.com/LawnchairLauncher/lawnicons/blob/develop/CONTRIBUTING.md) and will make changes if someone suggests. I will also make sure that Lawnicons builds correctly.

### CODE REVIEW

**Review by Ahmed-Khalifa — COMMENTED** [review]
Kindly approve.
Can you help me with how to use "git reset --hard" command if you have time to explain?

**Comment by I21b** [comment]
> Kindly approve. Can you help me with how to use "git reset --hard" command if you have time to explain?

https://git-scm.com/docs/git-reset

File: svgs/teampixel.svg
Comment (x9136) [inline svgs/teampixel.svg:1]: Screenshot 2024-06-15 at 00 08 42
The icon size is slightly smaller than needed.

It is also worth raising the icon 1px higher to compensate for the emptiness at the top.

File: app/assets/appfilter.xml
Comment (x9136) [inline app/assets/appfilter.xml:1]: drawable="teampixel" → drawable="team_pixel_wallpapers"
teampixel.svg → team_pixel_wallpapers.svg

**Review by x9136 — CHANGES_REQUESTED** [review]

**Comment by x9136** [comment]
> Can you help me with how to use "git reset --hard" command if you have time to explain?

As far as I remember, this topic helped me: https://stackoverflow.com/questions/50112363/how-to-fix-your-branch-is-ahead-of-origin-master-by-1-commit.

My process is like this:
1. I'm doing "Fetch origin" in GitHub Desktop.
2. I open the console and write `git reset --hard upstream/develop`. As far as I understand, this allows me to synchronize the main repository (LawnchairLauncher:develop) with my local copy.
3. After that, I do "Force push origin" in GitHub Desktop. This allows me to replace the content of my branch with the content of my local copy. As a result, my branch turns out to be the same as the main repository.

Perhaps I also configured through the console what GitHub Desktop will consider the main repository. But it's kind of easy to Google.

**Pull request merged**